llama-index-integrations/llms/llama-index-llms-openai/llama_index/llms/openai/base.py

```diff
@@ -264,6 +265,7 @@ def __init__(
         self._aclient = None
         self._http_client = http_client
         self._async_http_client = async_http_client
+        self.strict = strict
```

Comment:
we should instead add `strict` as a class attribute (like api_key, default_headers, etc. above) and then pass `strict` into `super().__init__(..., strict=strict)`